src/main/resources/schemas/opa_press_list.json

```diff
@@ -0,0 +1,600 @@
+{
```

Comment:
I've done a comparison of this schema vs what I can see used in the Style Guide ticket.

I believe the following are unused:

- Document Name
- Venue Email + Telephone
- Court House name / Court room name potentially? (Unsure where the heading above Hide All Sections should come from)
- Unable to see 'youth' in the style guide

Reply:
Will need to check with Chris C tomorrow.

Reply:
Discussed with Chris C. He said keeping some of the unused fields but removed some. The following fields have been removed:
- courtHouseName
- courtRoomName
- sittingStart
- friendlyRoleName
- title

I have also added 2 new fields to the offence level
- individualDetails > offence > reportingRestrictionDetail
- organisationDetails > offence > reportingRestrictionDetail